--- bu_Makefile	2020-05-08 19:26:21.000000000 +0900
+++ Makefile	2020-05-08 19:26:58.000000000 +0900
@@ -1,4 +1,4 @@
-PREFIX  = /usr/local
+PREFIX  = ~/12step/tools
 ARCH    = h8300-elf
 BINDIR  = $(PREFIX)/bin
 ADDNAME = $(ARCH)-
@@ -13,11 +13,11 @@ OBJDUMP = $(BINDIR)/$(ADDNAME)objdump
 RANLIB  = $(BINDIR)/$(ADDNAME)ranlib
 STRIP   = $(BINDIR)/$(ADDNAME)strip
 
-H8WRITE = ../../tools/h8write/h8write
+H8WRITE = ../../../tools/kz_h8write
 
 # FreeBSD-4.x:/dev/cuaaX, FreeBSD-6.x:/dev/cuadX, FreeBSD(USB):/dev/cuaUx
 # Linux:/dev/ttySx, Linux(USB):/dev/ttyUSBx, Windows:comX
-H8WRITE_SERDEV = /dev/cuad0
+H8WRITE_SERDEV = /dev/tty.usbserial-FTSXFJAF
 
 OBJS  = vector.o startup.o main.o
 OBJS += lib.o serial.o
